### ISSUE TITLE
[Bug] fenix opens github link with mgit

### ISSUE THREAD

**Issue by 532910** [mozilla-mobile/fenix#8587]
## Steps to reproduce
Open a page with links to github and press github link:
![image](https://user-images.githubusercontent.com/14617699/74955651-0221ba80-5416-11ea-9f3a-fad257f6c64c.png)

### Expected behavior
Link should be opened in the current tab

### Actual behavior
Link is opened in mgit

Both fenix and mgit has no defaults set:
![image](https://user-images.githubusercontent.com/14617699/74955786-34331c80-5416-11ea-9aeb-5407ed9d1c6f.png)
![image](https://user-images.githubusercontent.com/14617699/74955813-3dbc8480-5416-11ea-8381-1a17c6be488c.png)

Fennec and fenix nightly open github links in the current tab as expected.

### Device information
Android 9 (LineageOS 16.0) OP3T
Fenix version: 3.2.1 (Build #20250350)

**Comment by 532910** [comment]
This behavior is due to `Open links in Apps` control, possibly not a bug, but odd.

**Comment by softvision-miralobontiu** [comment]
@532910 does this behavior still occurs on the latest Firefox Nightly?
Thank you!

**Comment by 532910** [comment]
Nope, the latest Nightly just shows the same link with `Open links in apps` toggled on. What is this setting for?

**Comment by sflorean** [comment]
@532910 the setting is used for the YouTube links, to be opened in YouTube app. This is an example, but depends on what apps have you installed on your device, and the toggle must be ON. 
I will close this based on the fact that this is no longer reproducible.
